frontend/src/draft-review.js

```diff
@@ -337,6 +349,52 @@ class DraftReview extends React.Component {
     }
   }
 
+  getChangedNodes = () => {
```

Comment:
nodes.map((node) => {
      if (node.data.get("diff_state") === 'diff-added' && lastDiffState !== 'diff-added') {
        keys.push(node.key);
      }
      else if (node.data.get("diff_state") === 'diff-removed' && lastDiffState !== 'diff-removed') {
        keys.push(node.key);
      }
      lastDiffState = node.data.get("diff_state");
    });
    this.setState({
      changedNodes: keys
    });
  }